internal/models/v2models/chart_release_version_test.go

```diff
@@ -97,7 +100,8 @@ func TestChartReleaseVersion_validate(t *testing.T) {
 				},
 				ChartVersionID: testutils.PointerTo[uint](2),
 
-				HelmfileRef: testutils.PointerTo("e5f6g7h8"),
+				HelmfileRef:         testutils.PointerTo("e5f6g7h8"),
+				FirecloudDevelopRef: testutils.PointerTo("dev"),
```

Comment:
if fc-dev ref is no longer a mandatory field I think we should omit it at some points in these validation tests to cover that case